docs/packages/dara-components/graphs/introduction.mdx

```diff
@@ -336,3 +402,29 @@ config.add_page(name='Skeleton Example', content=CausalGraphViewer(causal_graph=
 ```
 
 ![CausalGraphViewer Skeleton Example](./assets/CausalGraphViewerSkeleton.png)
+
+### TimeSeriesCausalGraph
+
+For [TimeSeriesCausalGraph](https://causalgraph.causalens.com/docs/generated/cai-causal-graph/docs/time_series_causal_graph) we have an added in support where for layouts that support tiers, they will use `time_lag` and `variable_name` to place these nodes within the same tier.
```

Comment:
Might be worth explaining the behaviour more, that e.g. passing explicit tier config will override the behaviour and this is just a default if a time series graph is detected and the layout supports it.